scout/bin.py

```diff
@@ -23,20 +24,29 @@ class SolvParser(object):
     def __init__(self):
         self.pool = solv.Pool()
 
+        zypper_refresh_failed = False
         for repofile in [ f for f in os.listdir(self.etcpath) if fnmatch(f, '*.repo') ]:
             try:
                 parser = SafeConfigParser()
                 parser.read( '%s/%s' % (self.etcpath, repofile) )
                 for name in parser.sections():
                     if parser.get(name, 'enabled') == '1':
-                        if not os.path.isfile(self.solvfile % name):
-                            os.system('zypper refresh')
+                        if not os.path.isfile(self.solvfile % name.replace('/', '_')) and not zypper_refresh_failed:
```

Comment:
Still is not python3, we need to wait a bit.

Reply:
Uhm, I tested with python 2.7 . did I do something wrong? Anyway, I guess I could test with a venv of older python

Reply:
Oh I see the issue. Is `isfile`:

```
Python 2.7.14 (default, Oct 12 2017, 15:50:02) [GCC] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import os.path
>>> os.path.is_file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'is_file'
>>> os.path.isfile
<function isfile at 0x7fcd1bf76f50>
>>> 
```